casestudies/load-testing/README.md

```diff
@@ -0,0 +1,70 @@
+# ACK Performance at Scale
+
+In this usecase, we would deploy 10K SNS Topics, 20K SQS Queues and 20K SNS Subscriptions i.e 50K resources at a time using SQS and SNS ACK Controllers to examine performance of ACK and other metrics.
+
+**Testing environment:**
+* EKS cluster
+* ACK SNS and AWS SQS Controller
+* Prometheus + Grafana Stack
+
+![](ack.drawio.png)
+
+Prometheus is configured to scrape ACK Controller metrics which will help us analyze the performance and other metrics like resource counts, reconciliation time etc.
+*The 50K resources would be deployed using a Helm Chart.*
+
+After installing the Helm chart, following were the observations:
```

Comment:
Which helm chart? the controllers or the resources? can you talk a bit more about the resource helm chart? maybe share the example here?

Reply:
@kumudhanrajan18 I think you can add the helm chart here along with the helm command to install it.